static/base.html

```diff
@@ -30,13 +30,16 @@
             <div class="modal-dialog modal-dialog-centered">
                 <div class="modal-content">
                     <div class="modal-body">
-                        <p>Restarting ankerctl...</p>
+                        <p id="modal-inner">Restarting ankerctl...</p>
```

Comment:
If I understand you correctly, we want to be able to use the modal generically, right?

In that case, shouldn't `modal-inner` be blank here? Or did I misunderstand what we're going for?

Reply:
Yes, theoretically it should be blank, I just left it as is for no real reason at all :D